.github/workflows/main.yaml

```diff
@@ -30,6 +30,15 @@ jobs:
           restore-keys: |
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+      - name: Install python dependencies
+        run: python -m pip install --upgrade pip
+      - run: pip install pyteal
+      - run: pip install pyyaml
```

Comment:
we should test our `pipenv` workflow. So install `pipenv`, then `pipenv sync` and finally `pipenv shell`.

Reply:
https://github.com/scale-it/algo-builder/pull/367/checks?check_run_id=2764586424
getting this error

Reply:
updated to use pipenv